dashboard/test/factories.rb

```diff
@@ -98,6 +102,14 @@
       birthday Time.zone.today - 10.years
     end
 
+    factory :student_of_admin do
+      after(:create) do |user|
+        section = create :section
+        section.users << create(:admin_teacher)
+        create(:follower, section: section, student_user: user)
+      end
+    end
```

Comment:
Is this a remnant from an old base (merge)? AFAIK, it was deleted in semi-recent past.